saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -90,6 +115,13 @@ def clean_input(cls, info: ResolveInfo, instance, data, **kwargs):
             cleaned_input["slug"] = slugify(slug)
         if stock_settings := cleaned_input.get("stock_settings"):
             cleaned_input["allocation_strategy"] = stock_settings["allocation_strategy"]
+        if order_settings := cleaned_input.get("order_settings"):
+            cleaned_input["automatically_confirm_all_new_orders"] = order_settings[
+                "automatically_confirm_all_new_orders"
+            ]
```

Comment:
Don't you need to use `.get` on the dict here? These two input fields are not required, so I think the user might provide only one, and then one of these assignments would break? Same in the update mutation.